### PR TITLE
Convert `tsx` codeblocks in README to `ts` because there's no React code in here

### PR DESCRIPTION
**Pull request by steveluscher**

### CODE REVIEW

**Comment by steveluscher** [comment]
* **#2703** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2703?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2702** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2702?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2701** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2701?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @steveluscher and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/library/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ If you build JavaScript applications on Solana, it’s likely you’ve worked wi
 
 Here’s an example of a common code snippet from `@solana/web3.js`:
 
-```tsx
```
Comment (steveluscher) [inline packages/library/README.md:21]: Were these like this for a reason, @buffalojoec, @lorisleiva, @mcintyre94?

**Review by lorisleiva — APPROVED** [review]

File: packages/library/README.md (hunk 1 above)
Comment (mcintyre94) [reply packages/library/README.md:21]: We're all too react pilled.

**Review by mcintyre94 — APPROVED** [review]

**Comment by steveluscher** [comment]
### Merge activity

* **May 10, 8:14 AM PDT**: @steveluscher started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2701).

**Pull request merged**